test/modules/DirectMethodSender/DirectMethodCloudSender.cs

```diff
@@ -45,12 +45,7 @@ sealed class DirectMethodCloudSender : DirectMethodSenderBase
             ulong directMethodCount,
             CancellationToken none)
         {
-            TimeSpan timeouts = Settings.Current.DirectMethodDelay;
```

Comment:
do we have a bug for this? if so, please update that bug with this proposed fix in what file and line number.